### ISSUE TITLE
Input for review of imported WikiData definitions - for various terms

### ISSUE THREAD

**Issue by rrovetto** [ESIPFed/sweet#210]
## Description

The imported WikiData definition in SWEET 'star', and others (below), does not match the intended subject matter (scope) of its SWEET modules. Recommendations below. Will add more observations as I find them. This is a review in the context of WikiData descriptions to identify obvious errors, not a thorough analysis. A more thorough semantic analysis would include suggested edits to a given definition, suggestions for a missing or replacement definition. 

**Context:** https://github.com/ESIPFed/sweet/pull/200 , https://github.com/ESIPFed/sweet/pull/205 , etc.
**Related:** Thus far, I've done a partial cursory review of some of the recently imported WikiData definitions, and have added observations/recommendations to the [provided ](https://docs.google.com/spreadsheets/d/1NLtlAecaSlAOVy5_1_IQraORaKemnWNhxF0UXHbB44M/edit?usp=sharing) spreadsheet offline for some of the terms. Will continue as able.

## What would you like to see changed?

**Recommendation**:  Delete WIkiData definition of 'star' (http://sweetontology.net/realm/AstroBody/Star)
**Reason**:  Mismatch between intended scope of the SWEET module, and the imported definition.
**Alternative WikiData definition**:  https://www.wikidata.org/wiki/Q523

**Recommendation**: Delete definition of 'satellite' (http://sweetontology.net/realm/AstroBody/Satellite)
**Reason**:  Mismatch between intended scope of the SWEET module, and the imported definition.
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q1297322

**Recommendation**: Delete WikiData definition of 'moon' (http://sweetontology.net/realm/AstroBody/Moon)
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q2537

**Recommendation**: Delete WikiData definition of 'planetary boundary' (http://sweetontology.net/realm/PlanetaryBoundary)
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q1757268

**Recommendation**: Delete WikiData definition of 'position' (http://sweetontology.net/propSpaceLocation/Position)
**Observation**: WikiData currently seems to have it synonymous with 'location' (https://www.wikidata.org/wiki/Q17334923)

**Recommendation**: Delete WIkiData definition of 'size'(http://sweetontology.net/propSpaceDistance/Size)
**Reason**: Mismatch between intended scope of the SWEET module, and the imported definition.
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q322481

**Recommendation**: Delete WIkiData definition of 'axis' (http://sweetontology.net/propSpaceDirection/Axis)
**Reason**: Mismatch between intended scope of the SWEET module, and the imported definition.
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q1783819

**Recommendation**: Delete WIkiData definition of 'attitude' (http://sweetontology.net/propSpaceDirection/Attitude)
**Reason**: Mismatch between intended scope of the SWEET module, and the imported definition.
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q2235286

**Recommendation**: Delete WIkiData definition of 'shape' (http://sweetontology.net/propSpace/Shape)
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q207961

**Recommendation**: Delete WIkiData definition of 'abyss' (http://sweetontology.net/realm/Ocean/Abyss)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.
**Comment**: the context of the SWEET module suggests intended meaning is related to the abyssal zone, abyssal plain, etc. of the ocean. 

**Recommendation**: Delete WIkiData definition of 'file' (http://sweetontology.net/reprDataFormat/File)
**Reason**: too specific, i.e. wrt Unix.
**Alternative WikiData def**: https://www.wikidata.org/wiki/Q82753

**Recommendation**: Delete WIkiData definition for 'data' (http://sweetontology.net/reprDataProduct/Data)
**Reason**: can be more precise.
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q494756

**Recommendation**: Delete WIkiData definition for 'process' (http://sweetontology.net/proc/Process)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition. 

**Recommendation**: Delete WIkiData definition for 'realm' (http://sweetontology.net/realm/Realm)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition. 

**Recommendation**: Delete WIkiData definition of 'bombardment' (http://sweetontology.net/procPhysical/Bombardment)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.

**Recommendation**: Delete WIkiData definition of 'arc' (http://sweetontology.net/realmGeolOrogen/Arc)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.

**Recommendation**: Delete WIkiData definition of 'substratum' (http://sweetontology.net/realmSoil/Substratum)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.

**Recommendation**: Delete WIkiData definition of 'structure' (http://sweetontology.net/reprDataModel/Structure)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.

**Recommendation**: Delete WIkiData definition of 'pole' (http://sweetontology.net/reprSpaceCoordinate/Pole)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.

**Recommendation**: Delete WIkiData definition of 'circular' (http://sweetontology.net/reprSpaceGeometry/Circular)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.
**Alternative WikiData definition**: https://www.wikidata.org/wiki/Q59564206

**Recommendation**: Delete WIkiData definition of 'sphere' (http://sweetontology.net/reprSpaceGeometry3D/Sphere)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.

**Recommendation**: Delete WIkiData definition of 'bottom' (http://sweetontology.net/stateSpaceConfiguration/Bottom)
**Reason**: Mismatch between scope of the SWEET module, and the imported definition.

**Recommendation**: Delete WIkiData definition of 'high tide' (http://sweetontology.net/stateTimeCycle/HighTide)
**Reason**: too specific. can be more precise.
**Alternative WikiData def**: https://www.wikidata.org/wiki/Q20631632

**Recommendation**: Delete WIkiData definition of 'era' (http://sweetontology.net/stateTime/Era)
**Reason**: too specific, if the SWEET module is not geo. Should be more general, e.g., as a reference time.
**Alternative WikiData def**: https://www.wikidata.org/wiki/Q2703

## Attribution
if needed, http://orcid.org/0000-0003-3835-7817

## Labeling your issues 
if needed, 'realm', 'enhancement'

**Comment by lewismc** [comment]
@rrovetto are you suggesting that we remove these `skos:definition`'s? I don't have an issue with that, I just want to clarify. Thanks

**Comment by rrovetto** [comment]
> 
> 
> @rrovetto are you suggesting that we remove these `skos:definition`'s? I don't have an issue with that, I just want to clarify. Thanks

Yes, for those I wrote 'delete [...]'--if the skos:definition annotation contains the Wikidata definition indicated in the spreadsheet you [provided](https://docs.google.com/spreadsheets/d/1NLtlAecaSlAOVy5_1_IQraORaKemnWNhxF0UXHbB44M/edit?usp=sharing). 

For example, the [wikidata definition](http://www.wikidata.org/entity/Q1420750) in the spreadsheet for the SWEET astroBody/Star (term = 'star') should be removed. 
If desired, I included an alternative WikiData def I found that more closely matches. If not, then the minimum recommendation is to remove.
Sorry if my wording in the issue post isn't clear. If you prefer, I can edit it to say 'remove current wikidata def in skos:def annotation', etc.
